networking/eth2/src/test-support/java/tech/pegasys/artemis/networking/eth2/Eth2NetworkFactory.java

```diff
@@ -80,6 +80,9 @@ protected Eth2Network buildAndStartNetwork() throws Exception {
         try {
           network.start().get(30, TimeUnit.SECONDS);
           networks.add(network);
+          for (Eth2Network peer : peers) {
+            network.connect(peer.getNodeAddress()).join();
+          }
```

Comment:
Why add explicit connection logic here instead of just configuring the static peers?

Reply:
With `Eth2Network` now holding a `DiscoveryNetwork` rather than the other way around we can go back to just using static peers.  Previously there was no `ConnectionManager` created so nothing actually used the static peers config.  Now that it's switched around it makes sense to create `DiscoveryNetwork` with a static discovery system and we can use the config again.  Good sign that having it this way round makes more sense.